maquetado/index.html

```diff
@@ -8,37 +8,41 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
+  <div class="navbar">
+    <img class="nav-logo" src="./assets/wolox_logo.svg" alt="">
```

Comment:
Add an alt attribute. Every image should have one

Reply:
Remember to add the alternative text.